Makefile

```diff
@@ -182,7 +181,7 @@ clean_docker:
 
 push_docker: docker
 	# Weird if logic is so we fail if all the pushes fail
-	for i in $$(seq 1 6); do if [[ $$i == "6" ]] ; then exit 1 ; fi ; docker push $(docker_image):$(TOIL_DOCKER_TAG) && break || sleep 60; done
+	cd docker ; for i in $$(seq 1 6); do if [[ $$i == "6" ]] ; then exit 1 ; fi ; docker buildx build --platform linux/amd64,linux/arm64 --push --tag=$(docker_image):$(TOIL_DOCKER_TAG) -f Dockerfile . && break || sleep 60; done
 	for i in $$(seq 1 6); do if [[ $$i == "6" ]] ; then exit 1 ; fi ; docker push $(grafana_image):$(TOIL_DOCKER_TAG) && break || sleep 60; done
 	for i in $$(seq 1 6); do if [[ $$i == "6" ]] ; then exit 1 ; fi ; docker push $(prometheus_image):$(TOIL_DOCKER_TAG) && break || sleep 60; done
 	for i in $$(seq 1 6); do if [[ $$i == "6" ]] ; then exit 1 ; fi ; docker push $(mtail_image):$(TOIL_DOCKER_TAG) && break || sleep 60; done
```

Comment:
Should we be building the metrics containers as ARM as well?

Reply:
We currently only use them on Mesos, and we have no Mesos vuild for ARM and therefore no ARM Mesos clusters.

Maybe if we decide to build out the dashboard on Kubernetes we'd need ARM versions of these images. Even then, the Right Way to do the dashboard on Kubernetes would probably be to mount all the Toil-specific config files and do all the Toil-specific customization at the Kubernetes level, on top of stock grafana, mtail, and prometheus images.

Reply:
I'll make an issue based on this then.